### ISSUE TITLE
Sigmoid output instead of Softmax for BertForSequenceClassification models

### ISSUE THREAD

**Issue by uzpaka** [JohnSnowLabs/spark-nlp#7137]
Hello,

I have finetuned a multi-label BertForSequenceClassification model with huggingface and successfully converted it to spark-nlp. However, the output logits of BertForSequenceClassification in spark-nlp are always passed to a softmax function:

https://github.com/JohnSnowLabs/spark-nlp/blob/8a5898d16f6b0fda4123deba5c920dff01e756c2/src/main/scala/com/johnsnowlabs/ml/tensorflow/TensorflowBertClassification.scala#L112

https://github.com/JohnSnowLabs/spark-nlp/blob/8a5898d16f6b0fda4123deba5c920dff01e756c2/src/main/scala/com/johnsnowlabs/ml/tensorflow/TensorflowBertClassification.scala#L160

https://github.com/JohnSnowLabs/spark-nlp/blob/8a5898d16f6b0fda4123deba5c920dff01e756c2/src/main/scala/com/johnsnowlabs/ml/tensorflow/TensorflowForClassification.scala#L116

The issue is that for multi-label, we need a sigmoid activation and not a softmax one. I would highly appreciate if you could add an option to choose between these 2 activation functions (or even get the raw logits). Since softmax is not invertible, I cannot recover the original logits and do the sigmoid myself on the output dataframe.

Note that multi-label is a valid argument for the original BertForSequenceClassification/TFBertForSequenceClassification model in huggingface (See https://huggingface.co/docs/transformers/model_doc/bert#transformers.BertForSequenceClassification under "Example of multi-label classification")

Thank you!

**Comment by maziyarpanahi** [comment]
Hi @uzpaka 

This is a very valid and good feature request. Perhaps we can have a param to allow users to decide the activation they need and leave it to default on `softmax`. We do have multi-class (ClassifierDL) and multi-label (MultiClassifierDL) annotators, so I am very well aware that this is a valid request.

I will look into `problem_type="multi_label_classification"` on HF and see if I can implement the `calcualteSigmoid` and implement that param for all the `ForSequenceClassification` annotators we have.

**Comment by maziyarpanahi** [comment]
Hi @uzpaka 

I am starting to implement this option in all the classifiers, could you please either share your model or share a link to a BertForSequenceClassification model on HuggingFace that was trained on a multi-label dataset? (I want to make sure the sigmoid is implemented correctly). Thank you

**Comment by maziyarpanahi** [comment]
@uzpaka 

This should be now available in 3.4.3 release: https://github.com/JohnSnowLabs/spark-nlp/releases/tag/3.4.3
